CONTRIBUTING.md

```diff
@@ -87,3 +87,30 @@ You can enable it in VSCode with the following in your `settings.json`.
     }
   },
 ```
+
+# Testing
+
+We believe automated tests ensure the integrity of all Wormhole components. Anyone can verify or extend them transparently and they fit nicely with our software development lifecycle. This ensures Wormhole components operate as expected in both expected and failure cases.
+
+Places to find out more about existing test coverage and how to run those tests:
+
+- **Guardian Node**
+    - Tests: `./node/**/*_test.go`
+    - Run: `cd node && make test`
+- **Ethereum Smart Contracts**
+    - Tests: `./ethereum/test/*.[js|sol]`
+    - Run: `cd ethereum && make test`
+- **Solana Smart Contracts**
+    - Tests: `./solana/bridge/program/tests/*.rs`
+    - Run: `cd solana && make test`
+- **Terra Smart Contracts**
+    - Tests: `./terra/test/*`
+    - Run: `cd terra && make test`
+- **Terra2 Smart Contracts**
```

Comment:
This is not just terra2, but the more general chain-agnostic cosmwasm contracts

Reply:
merging just to get this live and not squander their reviews, will send a minor edit for this shortly.

Reply:
https://github.com/certusone/wormhole/pull/1424